sdk/src/main/java/io/opentelemetry/sdk/trace/ReadableSpan.java

```diff
@@ -60,4 +60,21 @@
    *     library
    */
   InstrumentationLibraryInfo getInstrumentationLibraryInfo();
+
+  /**
+   * Returns whether this Span has already been ended.
+   *
+   * @return {@code true} if the span has already been ended, {@code false} if not.
+   * @since 0.4.0
+   */
+  boolean hasEnded();
+
+  /**
+   * Returns the latency of the {@code Span} in nanos. If still active then returns now() - start
```

Comment:
I think it would be nice in this javadoc to define "latency" in this context, because it is a word that some may interpret as meaning "network latency", which seems like an odd thing to have on a generic Span.

Reply:
Actually I think it might be better to just rename this to `currentDuration`. Latency, while technically correct (see https://github.com/open-telemetry/opentelemetry-specification/issues/330), is probably confusing to most users.

Reply:
Or maybe "elapsedTime" ?